src/SingleStoreConnector/Core/ServerSession.cs

```diff
@@ -1035,7 +1039,11 @@ private async Task<bool> OpenTcpSocketAsync(ConnectionSettings cs, ILoadBalancer
 						{
 							if (ioBehavior == IOBehavior.Asynchronous)
 							{
+#if NET5_0_OR_GREATER
+								await tcpClient.ConnectAsync(ipAddress, cs.Port, cancellationToken).ConfigureAwait(false);
+#else
 								await tcpClient.ConnectAsync(ipAddress, cs.Port).ConfigureAwait(false);
+#endif
```

Comment:
Suggesting this change on original MySqlConnector here https://github.com/mysql-net/MySqlConnector/pull/1291